frontend/packages/policy-editor/src/components/ExpandablePolicyCard/ExpandablePolicyCard.tsx

```diff
@@ -575,7 +514,7 @@ export const ExpandablePolicyCard = ({
         {showErrors &&
           hasSubjectsError &&
           displayWarningCard(t('policy_editor.rule_card_subjects_error'))}
-        <Label className={classes.label} size='medium'>
+        <Label as='p' className={classes.label} size='medium'>
```

Comment:
Er det noen grunn til å bruke `as='p'` her? Med `label` kan man bruke `for`-attributtet (`htmlFor` på Reactsk) til å angi hvilket felt labelen gjelder slik at brukeren kan klikke på den for å fokusere på feltet. I tillegg slipper man da å bruke `aria`-attributter.

Reply:
Fixed